README.md

```diff
@@ -181,6 +181,20 @@ details.
 We welcome contribution from everyone. Read more about it in
 [`CODE_OF_CONDUCT.md`](https://github.com/envato/jwt_signed_request/blob/master/CODE_OF_CONDUCT.md)
 
+## Contributers
+
+Many thanks to the following contributers to this gem:
+
+- Toan Nguyen - [@yoshdog](https://github.com/yoshdog)
+- Odin Dutton - [@twe4ked](https://github.com/twe4ked)
+- Sebastian von Conrad - [@vonconrad](https://github.com/vonconrad)
+- Zubin Henner- [@zubin](https://github.com/zubin)
+- Glen Tweedie - [@nocache](https://github.com/nocache)
+- Giancarlo Salamanca - [@giancarlo](https://github.com/giancarlo)
```

Comment:
Nitpick - I'm actually @salamagd on Github but I can change this myself 😉

Reply:
😆 that may be easier